lib/vachan_web/live/verify_email_live.ex

```diff
@@ -7,9 +7,16 @@ defmodule VachanWeb.VerifyEmailLive do
 
   def render(assigns) do
     ~H"""
-    <div class="flex justify-center mt-12">
-      <h1>Verify Email</h1>
-      You must verify your email.
+    <div class="flex justify-center items-center min-h-screen" style="background-color: rgb(237, 237, 208);">
+      <div class="bg-white p-8 rounded shadow-md text-center max-w-md w-full">
```

Comment:
- update background color to bg-customBackground_header  instead of bg-white.
- alignment should be justify ,it will look more better than center.

Reply:
- Okay, I have updated the bg colour. 
- I have put the alignment to justify centre only, not sure what do you mean by just "justify"

Reply:
> * Okay, I have updated the bg colour.
> 
>     * I have put the alignment to justify centre only, not sure what do you mean by just "justify"

just put text-justify intead of text-center

Reply:
Done, have a look.

Reply:
looks good